packages/core/src/sass/globals.scss

```diff
@@ -47,7 +47,7 @@ $uikit-maxwidth: 42em; //Answer to life, the universe, and everything (keeps thi
 $uikit-colour-Hero: #115361 !global; //color variables (yes! the actual spelling doesn't have a u)
 $uikit-colour-Primary: #880e48 !global;
 $uikit-colour-Background: mix(darken(#f0f3f5, 40), #f0f3f5, 50);
-$uikit-colour-Border: mix(black, #bebebe, 50) !global;
+$uikit-colour-Border: #bbbbbb !global;
```

Comment:
Hi Elise,

No all values in `core` need to be `global`. And all other exports per module for that matter from the global files.
The idea is that those colors and mixins ARE the design system. If/when we have to change the design for the gov.au system, this is where that would happen. Giving developers the ability to overwrite those changes means we don't have a system at all. We keep those globals to a minimum but `core` only has globals as that's it's nature.